hbase-common/src/main/java/org/apache/hadoop/hbase/io/compress/DictionaryCache.java

```diff
@@ -91,7 +91,7 @@ public byte[] load(String s) throws Exception {
     try {
       return CACHE.get(path);
     } catch (ExecutionException e) {
-      throw new IOException(e);
+      throw new RuntimeException("Unable to load dictionary at " + path, e);
```

Comment:
Why change the exception type? Is this not _necessarily_ an IOException?

Reply:
I did this to be more consistent with ZstdCodec line 153. At some point in the stack we need to swallow a checked exception or make it unchecked.

Reply:
👍